app/code/Magento/CmsGraphQl/etc/schema.graphqls

```diff
@@ -15,7 +15,8 @@ type Query {
         id: Int @doc(description: "Id of the CMS page")
     ): CmsPage @resolver(class: "Magento\\CmsGraphQl\\Model\\Resolver\\Page") @doc(description: "The CMS page query returns information about a CMS page") @cache(cacheTag: "cms_p", cacheIdentity: "Magento\\CmsGraphQl\\Model\\Resolver\\Page\\Identity")
     cmsBlocks (
-        identifiers: [String] @doc(description: "Identifiers of the CMS blocks")
+        identifiers: [String] @doc(description: "Identifiers of the CMS blocks"),
+        scope_code: [String] @doc(description: "Scope Code of the CMS blocks")
```

Comment:
Case: send
```graphql
query {
  storeConfig {
    id
    website_id
  }
  cmsBlocks(
    identifiers: null
    scope_code: null
  ) {
    items {
      content
    }
  }
}
```
and get this `"message": "\"identifiers\" of CMS blocks should be specified"`
send another
```graphql
query {
  storeConfig {
    id
    website_id
  }
  cmsBlocks(
    identifiers: ""
    scope_code: null
  ) {
    items {
      content
    }
  }
}
```
and get "message": "Scope Code of CMS blocks should be specified"
then send
```graphql
query {
  storeConfig {
    id
    website_id
  }
  cmsBlocks(
    identifiers: ""
    scope_code: ""
  ) {
    items {
      content
    }
  }
}
```
and get "The CMS block with the \"\" ID doesn't exist."

@naydav, why `identifiers` is not required field. What is the point of sending empty strings. I have seen many times that sending "" or null results in error that some field is not-nullable. Maybe the better logic is to check if the property is omitted and thn address it to `all` or `default` equivalent in REST rather than allowing empty strings?